addons/account_edi_proxy_client/models/account_edi_proxy_user.py

```diff
@@ -38,6 +38,7 @@ class AccountEdiProxyClientUser(models.Model):
     company_id = fields.Many2one('res.company', string='Company', required=True,
         default=lambda self: self.env.company)
     edi_identification = fields.Char(required=True, help="The unique id that identifies this user, typically the vat")
+    edi_identification_aliases = fields.Json(help="Alternative IDs to identify the proxy user in localized modules")
```

Comment:
- This is stable, there can't be any DB changes, so this fields needs to be a computed non-stored. 
- I would even remove the field on Odoo, and just keep the function.
  We have VAT and Codice Fiscale already, to hold the information. 
  On the IAP it's different because we don't have a VAT for the registered Proxy user there yet.